### PR TITLE
actions: add pylint.yml

### PR DESCRIPTION
**Pull request by kobryl**

### CODE REVIEW

**Comment by kobryl** [comment]
Merge only after #19 (first Python on master branch)

File: .pylintrc [hunk 1]
```diff
@@ -0,0 +1,648 @@
...
+max-public-methods=20
+
+# Maximum number of return / yield for function / method body.
+max-returns=6
+
+# Maximum number of statements in function / method body.
+max-statements=50
+
+# Minimum number of public methods for a class (see R0903).
+min-public-methods=2
```
Comment (michalkr52) [inline .pylintrc:320]: Couldn't this potentially cause problems when making simple prototypes/skeletons of classes?

File: .pylintrc [hunk 2]
```diff
@@ -0,0 +1,648 @@
...
+# Allow the body of an if to be on the same line as the test if there is no
+# else.
+single-line-if-stmt=no
+
+
+[IMPORTS]
+
+# List of modules that can be imported at any level, not just the top level
+# one.
+allow-any-import-level=
```
Comment (michalkr52) [inline .pylintrc:363]: I've seen and even once written such imports
Example: [importing blueprints in a Flask app factory according to docs](https://flask.palletsprojects.com/en/3.0.x/patterns/appfactories/#basic-factories)

File: objects_counter/api/results/views.py
Comment (michalkr52) [inline objects_counter/api/results/views.py:1]: You've forgotten to edit the attribute name in `@api.doc` string

**Review by michalkr52 — CHANGES_REQUESTED** [review]
I left two comments about pylint, it's up to you if you address them or not.
The rest of pylint seems fine, but there's a mistake in `views.py`. Correct me if I'm wrong.

File: .pylintrc (hunk 1 above)
Comment (kobryl) [reply .pylintrc:320]: You can disable it locally

File: .pylintrc (hunk 2 above)
Comment (kobryl) [reply .pylintrc:363]: You can disable it locally if necessary

**Review by michalkr52 — APPROVED** [review]
lgtm

**Pull request merged**